src/Console/Output/Progress/ProgressOutputFactory.php

```diff
@@ -45,9 +47,18 @@ public function create(string $outputType, OutputContext $context): ProgressOutp
             );
         }
 
+        if (ProgressOutputType::NONE === $outputType) {
+            return new NullOutput();
+        }
+
         return new self::$outputTypeMap[$outputType]($context);
     }
 
+    public function supports(string $outputType): bool
+    {
+        return $this->isBuiltInType($outputType);
```

Comment:
Why keeping the `isBuiltInType` method?

Reply:
This is leftover from initial implementation which has also support for custom outputs, so `supports()` = `isBuiltIn()` + `isCustom()`. Also today's rebase was done "more or less", as some changes from this PR were extracted when the progress bar was introduced.

Do you think it should be removed now, and re-introduced when needed (custom progress outputs support)?

Reply:
I would remove it for now. Without this context, `supports()` is just a public alias for a private method and there doesn't seem to be a logic reason for that.